go/src/koding/klientctl/machine.go

```diff
@@ -256,6 +256,42 @@ func MachineCpCommand(c *cli.Context, log logging.Logger, _ string) (int, error)
 	return 0, nil
 }
 
+// MachineInspectMountCommand allows to inspect internal mount status.
+func MachineInspectMountCommand(c *cli.Context, log logging.Logger, _ string) (int, error) {
+	// Machine inspect command needs exactly one identifier. Either mount ID or
+	// mount local path.
+	idents, err := getIdentifiers(c)
+	if err != nil {
+		return 1, err
+	}
+	if err := identifiersLimit(idents, "mount", 1, 1); err != nil {
+		return 1, err
+	}
+
+	// Enable all options when none of them are set.
+	isSync := c.Bool("sync")
+	if !isSync {
+		isSync = true
+	}
+
+	opts := &machine.InspectMountOptions{
+		Identifier: idents[0],
+		Sync:       isSync,
```

Comment:
for now - yes, I'm planning to add other options so isSync won't be always true.